packages/react-integration/demo-app-ts/src/components/demos/LabelGroupDemo/LabelGroupWithCategoryDemo.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
```

Comment:
Can you add the new `isOverflowLabel` to the demo app.
Can you also make sure all the props for the `LabelGroup` are used at least once in the demo to verify typing.